### PR TITLE
Update futubull from 12.2.2118 to 12.4.2308

### PR DESCRIPTION
**Pull request by BleedingSky**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/futubull.rb [hunk 1]
```diff
@@ -11,10 +14,10 @@
...
     url "https://www.futunn.com/download/history?client=11"
     strategy :page_match do |page|
-      match = page.match(/FTNNForMac[._-]v?(\d+(?:\.\d+)+)[_-](\d+)[._-]Website.*?\.dmg/i)
+      match = page.match(/FTNNForMac[._-]v?(\d+(?:\.\d+)+)[._-]Website.*?\.dmg/i)
       next if match.blank?
 
-      "#{match[1]},#{match[2]}"
+      (match[1]).to_s
     end
   end
```
Comment (bevanjkay) [inline Casks/futubull.rb:22]: @BleedingSky The livecheck block can be simplified.

```suggestion
  livecheck do
    url "https://www.futunn.com/download/history?client=11"
    regex(/FTNNForMac[._-]v?(\d+(?:\.\d+)+)[._-]Website\.dmg/i)
  end
```

File: Casks/futubull.rb [hunk 2]
```diff
@@ -1,8 +1,11 @@
+# typed: false
+# frozen_string_literal: true
+
```
Comment (bevanjkay) [inline Casks/futubull.rb:3]: @BleedingSky Please remove these three lines

**Review by bevanjkay — CHANGES_REQUESTED** [review]
@BleedingSky The livecheck block can be simplified. In future can you please open PRs from a branch on your fork (not your master/main branch). This will allow maintainers the ability to push changes to your PR.

**Comment by BleedingSky** [comment]
@bevanjkay Thank you for your comment.

**Pull request closed without merging**